src/test/java/htsjdk/samtools/util/IntervalListTest.java

```diff
@@ -50,9 +51,9 @@
 
     final SAMFileHeader fileHeader;
     final IntervalList list1, list2, list3;
-
+    static final public File TEST_DIR=new File("src/test/resources/htsjdk/samtools/intervallist");
     public IntervalListTest() {
-        fileHeader = IntervalList.fromFile(new File("src/test/resources/htsjdk/samtools/intervallist/IntervalListchr123_empty.interval_list")).getHeader();
+        fileHeader = IntervalList.fromFile(new File(TEST_DIR, "/IntervalListchr123_empty.interval_list")).getHeader();
```

Comment:
It would be nice to migrate this to `Path`, e.g. make `TEST_DIR` use `Path` and use `TEST_DIR.resolve("/xxx/xxx/")` here
```suggestion
        fileHeader = IntervalList.fromPath(TEST_DIR.resolve("/IntervalListchr123_empty.interval_list")).getHeader();
```